core/unit_test/TestLocalDeepCopy.hpp

```diff
@@ -907,7 +907,12 @@ void impl_test_local_deepcopy_rangepolicy_rank_7(const int N) {
 #if defined(KOKKOS_ENABLE_CXX11_DISPATCH_LAMBDA)
 TEST(TEST_CATEGORY, local_deepcopy_teampolicy_layoutleft) {
   using ExecSpace = TEST_EXECSPACE;
-  using ViewType  = Kokkos::View<double********, Kokkos::LayoutLeft, ExecSpace>;
+#if defined(KOKKOS_ENABLE_CUDA) && \
+    defined(KOKKOS_COMPILER_NVHPC)  // FIXME_NVHPC 23.7
+  if (std::is_same_v<ExecSpace, Kokkos::Cuda>)
+    GTEST_SKIP() << "FIXME_NVHPC (?)";
```

Comment:
The custom skip message is not great.

Reply:
Copied from elsewhere but I can update. Need to anyway fix OpenACC build